syntax = "proto3";

package whisper;

// ðŸ‘‡ Add this line
option go_package = "./proto;pb";

service WhisperService {
	rpc StreamAudio (stream AudioChunk) returns (Transcription);
}

message AudioChunk {
	bytes data = 1;
}

message Transcription {
	string text = 1;
}
